x-pack/plugins/cases/public/components/markdown_editor/plugins/lens/modal_container.tsx

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import styled from 'styled-components';
+
+export const ModalContainer = styled.div`
+  width: ${({ theme }) => theme.eui.euiBreakpoints.m};
+
+  .euiModalBody {
```

Comment:
Nitpick: I think it's better not to add styles to classes from EUI, but add a custom class coming from the plugin to the EUI element and add styles to this class. 
For example in `x-pack/plugins/cases/public/components/markdown_editor/plugins/lens/plugin.tsx` line 344:
`<EuiModalBody className="casesMarkdownModalBody">`

and then here targetting this class.